catroid/src/main/res/values/arrays.xml

```diff
@@ -203,5 +203,57 @@
         <item>@string/brick_set_physics_object_type_no</item>
     </string-array>
 
-
+    <!--Multilingual activity  -->
+    <string-array name="Languages_items" translatable="false">
```

Comment:
You could replace this array with an array full of country codes. Then in the code you can generate this array (for the adapter) from the country code array. This can be done by creating Locale objects with the country codes and using "getDisplayCountry()".
Also you should name it in lower case seperated with underscores.